aws/resource_aws_acm_certificate.go

```diff
@@ -60,11 +60,9 @@ func resourceAwsAcmCertificate() *schema.Resource {
 				},
 			},
 			"subject_alternative_names": {
-				Type:          schema.TypeList,
-				Optional:      true,
-				Computed:      true,
```

Comment:
Can you please leave the `Computed: true` field set for this attribute? This is here due to our `ImportCertificate` API support in this resource. Thanks!

Reply:
ACK